cts/cts-cli.in

```diff
@@ -538,6 +540,144 @@ function test_crm_mon() {
     unset CIB_file
 }
 
+function test_error_codes() {
+    # Note: At the time of this writing, crm_error returns success even for
+    # unknown error codes. We don't want to cause a regression by changing that.
+
+    # Legacy return codes
+    #
+    # Don't test unknown legacy code. FreeBSD includes a colon in strerror(),
+    # while other distros do not.
+    desc="Get legacy return code"
+    cmd="crm_error -- 201"
+    test_assert $CRM_EX_OK 0
+
+    desc="Get legacy return code (XML)"
+    cmd="crm_error --output-as=xml -- 201"
+    test_assert_validate $CRM_EX_OK 0
+
+    desc="Get legacy return code (with name)"
+    cmd="crm_error -n -- 201"
+    test_assert $CRM_EX_OK 0
+
+    desc="Get legacy return code (with name) (XML)"
+    cmd="crm_error -n --output-as=xml -- 201"
+    test_assert_validate $CRM_EX_OK 0
+
+    desc="Get multiple legacy return codes"
+    cmd="crm_error -- 1 2"
```

Comment:
errno's and their descriptions vary by OS, so we should only test our custom codes.

That will be a problem for the full list, not sure of the best approach there. Maybe |head -something or |grep something to spot-check.